modules/programs/ssh/default.nix

```diff
@@ -73,7 +73,7 @@ in
     };
   };
 
-  config = {
+  config = mkIf (builtins.length knownHosts > 0) {
```

Comment:
Thanks for the contribution, could you rebase this PR and move this check to just affect `/etc/ssh/ssh_known_hosts`?

Reply:
I'm sorry I dropped the ball on this one :( I've updated my PR and included a test.